app/assets/javascripts/application.js

```diff
@@ -25,6 +25,9 @@
 //= require controllers/ops/diagnostics_database_form_controller
 //= require controllers/ops/log_collection_form_controller
 //= require controllers/ops/tenant_form_controller
+//= require controllers/container_topology/container_topology_controller
+//= require d3/d3
```

Comment:
Right, missed that, sorry :) `d3` could be enough though, but c3 is as `c3/c3` as well, so ignore me :)